MediaGalleryUi/view/adminhtml/web/js/directory/directoryTree.js

```diff
@@ -96,10 +104,17 @@ define([
          * Verify directory filter on init event, select folder per directory filter state
          */
         checkChipFiltersState: function () {
-            if (!_.isUndefined(this.filterChips().filters.path) && this.filterChips().filters.path !== '') {
+            var currentFilterPath = this.filterChips().filters.path,
+                currentTreePath = Base64.decode(window.MediabrowserUtility.pathId.replace(/--|,,/, ''));
```

Comment:
We probably need to perform the same change done on the magento 2 PR, decode the pathId using a similar method that PHP does.

Reply:
that is not required when we decode value